src/game/server/entities/character.cpp

```diff
@@ -2918,6 +2912,12 @@ bool CCharacter::TakeDamage(vec2 Force, int Dmg, int From, int Weapon, int Mode)
 			}
 		}
 		
+		char aBuf[256];
+		str_format(aBuf, sizeof(aBuf), "kill killer='%s' victim='%s' weapon=%d",
+			Server()->ClientName(From),
+			Server()->ClientName(m_pPlayer->GetCID()), Weapon);
+		GameServer()->Console()->Print(IConsole::OUTPUT_LEVEL_DEBUG, "game", aBuf);
```

Comment:
i understand that the code was just moved, but the question about output level still stands

Reply:
Adding to my answer above: If your intention may be to hide these logs from the console, it would be via `console_output_level 1`